roqoqo-aqt/tests/integration/backend.rs

```diff
@@ -121,8 +689,8 @@ fn test_full_simple_gate() {
             200,
         );
 
-        let device = SimulatorDevice::new(2);
-        let backend = Backend::new(device.into(), None).unwrap();
+        let device = AqtDevice::new(2);
+        let backend = Backend::new(device, None).unwrap();
```

Comment:
I tried removing the `ignore` and the assert didn't pass.

Reply:
This seems to work usually, it rarely fails for me.